app/controllers/api/v0/articles_controller.rb

```diff
@@ -16,13 +16,17 @@ class ArticlesController < ApiController
       skip_before_action :verify_authenticity_token, only: %i[create update]
 
       def index
-        @articles = ArticleApiIndexService.new(params).get
+        @articles = ArticleApiIndexService.new(params, INDEX_ATTRIBUTES_FOR_SERIALIZATION).get
```

Comment:
as the selection is performed inside `ArticleApiIndexService` is done inside the service I need to get the attributes inside the class.

In theory I could keep it out by reworking the class a bit, let me know what you think

Reply:
Let's do it! I'm going to update the PR later